util/libcrypto.num

```diff
@@ -4417,3 +4417,11 @@ RAND_POOL_add                           4361	1_1_1	EXIST::FUNCTION:
 RAND_POOL_add_begin                     4362	1_1_1	EXIST::FUNCTION:
 RAND_POOL_add_end                       4363	1_1_1	EXIST::FUNCTION:
 RAND_POOL_acquire_entropy               4364	1_1_1	EXIST::FUNCTION:
+SM4_encrypt                             4365	1_1_1	EXIST::FUNCTION:SM4
+SM4_decrypt                             4366	1_1_1	EXIST::FUNCTION:SM4
+SM4_set_key                             4367	1_1_1	EXIST::FUNCTION:SM4
+EVP_sm4_ecb                             4368	1_1_1	EXIST::FUNCTION:SM4
+EVP_sm4_cbc                             4369	1_1_1	EXIST::FUNCTION:SM4
+EVP_sm4_ofb                             4370	1_1_1	EXIST::FUNCTION:SM4
+EVP_sm4_cfb128                          4371	1_1_1	EXIST::FUNCTION:SM4
+EVP_sm4_ctr                             4372	1_1_1	EXIST::FUNCTION:SM4
```

Comment:
I suggest you remove these changes and do a `make update` again.  You'll find that the `SM4_*` symbols will not appear...  (that's the update of 3 symbols that's mentioned in Travis, `make update` wants to change their state to `NOEXIST`)

Reply:
@levitte I've done this. Somehow `make update` on my machine adds a last line of `license ...  EXIST::FUNCTION` to the end, which I've removed manually... Let me know if this behavior is correct.

Reply:
I have no idea why it adds a `license` line...  It shouldn't, and it doesn't for me...

Reply:
Thanks @levitte for the confirmation 👍